load-testing/audit-service/sendLoginMessages.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Populates the SQS queue with couple of 100 messages for audit service load testing
+#
+
+export AWS_DEFAULT_REGION=us-east-1
+
+awsURL='https://sqs.us-east-1.amazonaws.com/707767160287'
+sqsURL='qaplanetv2--qa-niaid--audit-sqs'
+message='
+{
+    "idp":"fence",
+    "request_url": "",
+    "status_code": 200,
+    "sub": 1,
+    "username": "user_loadtest",
+}'
```

Comment:
you could add a parameter to your script to choose the message type, instead of duplicating the code in `sendLoginMessages.sh` and `sendPresignedURLMessages.sh`